app/src/main/res/values/ic_launcher_background2.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
too much xml files. also pay attention how it was named. ic_ means it is icon but you used it for color resources. You can create simple colors.xml file (or whatever you want) and use it for all colors, don't need to split such things.